tests/acceptance/features/cliProvisioning/userSync.feature

```diff
@@ -38,4 +38,26 @@ Feature: sync a user using occ command
       """
       OC\User\Database
       OCA\User_LDAP\User_Proxy
-      """
\ No newline at end of file
+      """
+
+  Scenario: sync a ldap user that is substring of some other ldap users
+    Given these users have been created with default attributes and without skeleton files:
+      | username    |
+      | regularuser |
+      | regular     |
+    When the administrator sets the ldap attribute "displayname" of the entry "uid=regular,ou=TestUsers" to "Test User"
+    And LDAP user "regular" is resynced
+    And user "regular" should exist
+    And user "regularuser" should exist
+    And the display name of user "regular" should be "Test User"
+
+  Scenario: sync a ldap user that is superstring of some other ldap users
```

Comment:
You need to `@skipOnOcV10.3`
The test scenarios are failing against `latest` 10.3.2

Reply:
done!